src/storage/src/source/postgres.rs

```diff
@@ -302,16 +303,16 @@ async fn postgres_replication_loop_inner(
                     &task_info.source_id, e
                 );
                 let reader = BufReader::new(writer.into_inner().into_std().await);
-                let res = PostgresTaskInfo::revert_snapshot(&mut snapshot_tx, reader)
+                // We don't close the `RowSender` on error, but an error on a `revert_snapshot` is
+                // not-recoverable and permanently errors the source.
+                // Additionally, <https://github.com/MaterializeInc/materialize/pull/13159>
```

Comment:
This is now merged, should this be removed?

Reply:
yes!